src/client/components/app/Navigation/NavSidebar.tsx

```diff
@@ -6,7 +6,7 @@ import { SettingsSelectors } from '@store';
 import { NavigationLink } from '@components/app';
 import { Link, Icon, Logo, RedirectIcon } from '@components/common';
 
-const linkHoverFilter = 'brightness(90%)';
+// const linkHoverFilter = 'brightness(90%)';
```

Comment:
If its no longer needed, lets remove it better :)

Reply:
ty